Clients/GUI/Pages/SignIn.razor

```diff
@@ -0,0 +1,71 @@
+﻿@page "/singin"
+@using Authentication
+@using ViewModels
+@using DTO
+@using Scripts
+@inject AuthStateProvider AuthenticationProvider
+@inject NavigationManager NavigationManager
+
+<h3 style="color: red; display: @(isHidden ? "none" : "block")"> Wrong login or password</h3>
```

Comment:
May be it could have been done in a more convenient way, but it's also fine

Reply:
But should it be above the input fields or under them?

Reply:
It usually done above the fields

Reply:
Ok, fine